File: board_dsl/src/main/java/com/dooho/board/dto/auth/SignInResponseDto.java
package com.dooho.board.dto.auth;

import com.dooho.board.entity.UserEntity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class SignInResponseDto {
    private String token;
    private Integer tokenExprTime;
    private String refreshToken;
    private Integer refreshExprTime;
    private UserEntity user;
}

File: board_spring/src/main/java/com/dooho/board/dto/auth/SignInResponseDto.kt
package com.dooho.board.dto.auth

import com.dooho.board.entity.UserEntity

data class SignInResponseDto (
    val token : String? = null,
    val tokenExprTime : Int? = null,
    val refreshToken : String? = null,
    val refreshTokenExprTime : Int? = null,
    val user : UserEntity? = null
)